boards/arm/nrf53/nrf5340-dk/configs/qspi_cpuapp/#defconfig#

```diff
@@ -21,14 +21,14 @@ CONFIG_BOARD_LOOPSPERMSEC=5500
 CONFIG_EXPERIMENTAL=y
 CONFIG_FS_LITTLEFS=y
 CONFIG_INIT_ENTRYPOINT="nsh_main"
+CONFIG_LINE_MAX=64
```

Comment:
what's this `#defconfig#` file?

Reply:
I have no idea

Reply:
let's wait for @raiden00pl's answer to https://github.com/apache/nuttx/pull/9387/files#r1915889205

Reply:
These are two different things. This PR is just to clear unnecessary definitions. If you think this defconfig file is unreasonable, please submit it by yourself after my PR is merged. Please don't block this PR process.

Reply:
IMO, we should block a PR making a modification which the author don't understand.